features/promotion/applying_catalog_promotions/applying_only_exclusive_catalog_promotion.feature

```diff
@@ -0,0 +1,26 @@
+@applying_catalog_promotions
+Feature: Applying only exclusive catalog promotion
+    In order to buy product with proper promotions
+    As a Visitor
+    I want to be able to set catalog promotion as exclusive
```

Comment:
Visitor is not able to set catalog promotion as exclusive one I guess, you should change the actor or adjust this feature description

Reply:
`I want to have only one exclusive catalog promotion applied on the product`?